primitives/src/constants.rs

```diff
@@ -42,8 +42,10 @@ pub const CENT: Balance = BASE / 100; // 100_000_000
 pub const MILLI: Balance = CENT / 10; //  10_000_000
 pub const MICRO: Balance = MILLI / 1000; // 10_000
 
+/// Storage cost per byte and item.
+// Approach: Achieve same cost per item and bytes in relation to total supply as on Polkadot.
 pub const fn deposit(items: u32, bytes: u32) -> Balance {
-    items as Balance * 20 * BASE + (bytes as Balance) * 100 * MILLI
+    items as Balance * 150 * CENT + (bytes as Balance) * 75 * MICRO
```

Comment:
It might be a good idea to achieve the same cost per item as on Polkadot in relation to the USD instead of the total supply :-D. Because now I the default cost on [Polkadot is 20 DOTs](https://github.com/paritytech/polkadot/blob/97930a8b2cc211ecfb80f289c4c578a9157e6dc4/runtime/polkadot/constants/src/lib.rs#L36-L38), so 100 USD, right? Here it is (without bytes) 1.5 ZTG, so 0.15 USD, right?

Reply:
There are two problems with your suggested approach:
1. It does not consider market cap. Polkadot is much more valuable, as it has a market cap of approx. 6 Billion USD right now. ZTG market cap is around approx. 5 Million USD. If Polkadot market cap would be the same as ZTG, you would also only pay roughly 0.07 USD. With current ZTG price, you also pay roughly 0.07 USD (1.5 * 5 cent). On the other hand, this means that as ZTG grows and becomes more valuable, the price for storage will also increase accordingly. If ZTG has a market cap of 6 Billion USD like Polkadot now, storage will also become more valuable (as the product is more valuable and utilized) and one storage item will cost also 100 USD. In addition to that, every ZTG already locked for storage deposit will also increase in value. So everyone who locks ZTG now for a storage item, locks only about 0.07 USD in value. But in the future, should ZTG reach a market cap like Polkadot, those ZTG will be worth 100 USD or even more.
2. If use the previous approach and charge ZTG worth 100 USD right now given the low market cap, we can only offer about 100 MB of storage space assuming that all the existing ZTG is only used to pay for storage space.